.github/workflows/pr.yml

```diff
@@ -547,7 +547,9 @@ jobs:
         uses: actions/setup-java@v4
         with:
           distribution: ${{ vars.VERSION_JAVA_DISTRIBUTION }}
-          java-version: ${{ vars.VERSION_JAVA }}
+          java-version: |
+            17
```

Comment:
we should add this version into `Repository variables` 
<img width="847" alt="image" src="https://github.com/user-attachments/assets/784262ef-aad5-4632-bc06-12f6196d998f">
This version is only needed for sdkmanager and we should not build with it the library

Reply:
This PR is preceded by https://github.com/realm/realm-kotlin/pull/1779. But the explicit reference to Java 17 is maintained. This syntax will cause `actions/setup-java` to install both `17` and `vars.VERSION_JAVA` so this is exactly allowing us to run the sdkmanager with Java 17 while building the rest with `vars.VERSION_JAVA` pointed to by `JAVA_HOME`.